bazel/bison.bzl

```diff
@@ -0,0 +1,65 @@
+"""Build rule for generating C or C++ sources with Bison."""
```

Comment:
I don't really know how Bazel works, but it would be nice if you didn't need to copy this file to every project using bison. Is there some kind of a (trusted) repository of such rules which would allow them to be imported dynamically?

Reply:
Agreed, that would be nice. At the moment, there is no trusted rule for Bison.